src/transformers/utils/generic.py

```diff
@@ -880,8 +880,8 @@ class TransformersKwargs(TypedDict, total=False):
     output_hidden_states: Optional[bool]
     output_attentions: Optional[bool]
     output_router_logits: Optional[bool]
-    cumulative_seqlens_q: Optional["torch.LongTensor"]
-    cumulative_seqlens_k: Optional["torch.LongTensor"]
+    cu_seq_lens_q: Optional["torch.LongTensor"]
+    cu_seq_lens_k: Optional["torch.LongTensor"]
```

Comment:
I don't mind this as in general we advertised that htese are unchanged and passed to fa2 directly

Reply:
I purpose the question in #40193, as I was misled by the `TransformersKwargs`and the finaly flash_attn_interface missed the `cumulative_seqlens_q/k` args. 

Actually, we still use `cu_seq_lens_q/k` in the `Inputs` and a lot of models create `cu_seq_lens_q/k` in Attention module.